Manifests/ManagedPreferencesApplications/com.google.Chrome.plist

```diff
@@ -15,7 +15,7 @@
 	<key>pfm_interaction</key>
 	<string>combined</string>
 	<key>pfm_last_modified</key>
-	<date>2023-08-15T08:00:00Z</date>
+  <date>2023-10-05T00:00:00Z</date>
```

Comment:
Line indentation shrunk.

Reply:
Fixed, sorry not sure how this happened.